bsp/qemu-sifive-e31/core.dts

```diff
@@ -161,7 +165,7 @@
                         reg = <0x10013000 0x1000>;
                         reg-names = "control";
                         clocks = <&hfclk>;
-                        pinmux = <&gpio0 0x30000 0x30000>;
+                        pinmux = <&gpio0 0x0 0x30000>;
```

Comment:
what is the reason for this change? previously incorrect?

Reply:
A change was made in gpio driver, to enable to set a bit in iof_sel register.
https://github.com/sifive/freedom-metal/blob/sifive-pwm/src/drivers/sifive_gpio0.c#L103
Previously it was, ''&=" operation, which wont allow to set a bit, if existing value is zero. 
As per GPIO Muxing for FE310, only PWM IP outputs are mapped to 'IOF1' everything else is mapped to 'IOF0'.
First value in this pinmux pair maps to 'source' parameter in __metal_driver_sifive_gpio0_enable_io() function. 
Corresponding values in dts need to be changed accordingly.